jsonrpc/jsonrpc.go

```diff
@@ -209,6 +210,11 @@ func (j *JSONRPC) handleWs(w http.ResponseWriter, req *http.Request) {
 				j.logger.Info("Closing WS connection with error")
 			}
 
+			ok := j.dispatcher.RemoveFilterByWs(wrapConn)
+			if !ok {
+				j.logger.Error("Unable to remove filter with specific WS")
+			}
+
```

Comment:
This can be inlined

Reply:
fixed dfa9cf9b1ee51761e111616bca079e4d33220df0